stglib/rsk/cdf2nc.py

```diff
@@ -229,6 +230,18 @@ def ds_add_attrs(ds):
             }
         )
 
+    def add_attributes(var, dsattrs):
+        var.attrs.update(
+            {
+                "initial_instrument_height": dsattrs["initial_instrument_height"],
```

Comment:
Do we need to set these on on every variable or can they remain as global atts??